rts/Game/UI/GuiHandler.cpp

```diff
@@ -3778,14 +3778,14 @@ void CGuiHandler::DrawMapStuff(bool onMiniMap)
 							}
 						}
 					}
-
-					if (unitDrawer->ShowUnitBuildSquare(bi, buildCommands)) {
-						glColor4f(0.7f, 1.0f, 1.0f, 0.4f);
-					} else {
-						glColor4f(1.0f, 0.5f, 0.5f, 0.4f);
+					if (drawBuildGrid || onMiniMap){
```

Comment:
because there are two cases of drawing : one drawing grids on the main window, the other on drawing grids on the minimap, the callin I made only disable/enable the drawing on the main window, it doesnt affect the drawing of grids on the minimap (I don't see any point of disabling it there)

Reply:
in other word, if I remove this condition it will also disable drawing grids on the minimap